src/pure/internal_pervasives.ml

```diff
@@ -277,6 +277,13 @@ module Display_markup = struct
     | None -> textf "%S" s
     | Some substr -> textf "%S... %d Bytes" substr (String.length s)
 
+  let big_itemize ?(max_display = 10) ~render l =
+    match List.length l with
+    | small when small <= max_display ->
+      itemize (List.map l ~f:render)
+    | big ->
```

Comment:
it's quite arbitrary to show the first ones Vs some other subset.

I consider that ones it's more than `max_display` one does not really care about the contents.